tests/test_form_fields.py

```diff
@@ -26,4 +26,5 @@ def test_bound_form_with_instance():
 
 def test_choices():
     form = get_form()
-    assert form.base_fields["zero2"].choices == [(0, 'ZERO'), (1, 'ONE')]
+    assert form.base_fields["zero2"].choices == [(0, 'Zero'), (1, 'One')]
+    assert form.base_fields["int_enum"].choices == BLANK_CHOICE_DASH + [(0, 'foo'), (1, 'B')]
```

Comment:
So, is this backward incompatible change, since the code for zero2 stays the same but still the choices change?